### PR TITLE
Conecting the layer parameters and geomswithAes

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons  i have published this now

File: instat/UcrGeomListWithAes.vb [hunk 1]
```diff
@@ -20,6 +20,7 @@ Public Class UcrGeomListWithParameters
     Public lstAesParameterUcr As New List(Of ucrReceiverSingle)
     Public lstCurrArguments As New List(Of String)
     Public bFirstLoad As Boolean = True
+    Public LayersControl As ucrLayerParameters
```
Comment (dannyparsons) [inline instat/UcrGeomListWithAes.vb]: You should prefix this with ucr

File: instat/ucrLayerParameters.vb [hunk 2]
```diff
@@ -16,6 +16,8 @@
 
 Public Class ucrLayerParameters
     Private bFirstLoad As Boolean = True
+    Public WithEvents GeomWithAes As UcrGeomListWithParameters
```
Comment (dannyparsons) [inline instat/ucrLayerParameters.vb]: Same prefix needed here.

**Comment by maxwellfundi** [comment]
@dannyparsons prefix added

**Comment by maxwellfundi** [comment]
@dannyparsons you were to look at it.

**Comment by dannyparsons** [comment]
Thanks

**Pull request merged**